src/js/groups/types.ts

```diff
@@ -1 +1,24 @@
+import { UserNested } from "../users/types";
+
 export type GroupMinimal = { id: string; name?: string };
+
+export type groupNameUpdate = {
+    id: string;
+    name?: string;
+};
```

Comment:
I think this is convention.

```suggestion
export type GroupNameUpdate = {
    id: string;
    name?: string;
};
```